web/onos-gui/src/app/onos-config/config-view/config-view.component.ts

```diff
@@ -39,23 +40,22 @@ export class ConfigViewComponent implements OnInit, OnChanges {
     type: string;
     updated: number = 0;
     changeIds: string[] = [];
-    changeIdsVisible: Map<string, boolean>;
-    hasStateData: boolean = true;
-    hasOpData: boolean;
+    changeIdsVisible = new Map<string, boolean>();
+    hasOpStateData: boolean = true;
+    create_pending: string = '';
+    create_pending_confirm: string = '';
 
     constructor(
         private diags: OnosConfigDiagsService,
+        private pending: PendingNetChangeService,
         protected ar: ActivatedRoute,
         protected is: IconService,
     ) {
         this.is.loadIconDef('checkMark');
         this.is.loadIconDef('xMark');
-        this.changeIdsVisible = new Map();
-        if (this.hasStateData) {
-            this.changeIdsVisible['state'] = false;
-        }
-        if (this.hasOpData) {
-            this.changeIdsVisible['operational'] = false;
+        this.is.loadIconDef('plus');
+        if (this.hasOpStateData) {
+            this.changeIdsVisible.set('opstate', false);
```

Comment:
worth extracting and re-using, otherwise easy to forget some place to change it

Reply:
done